internal/injector/builtin/yaml/databases/gocql.yml

```diff
@@ -0,0 +1,25 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-present Datadog, Inc.
+---
+# yaml-language-server: $schema=../../../../../docs/static/schema.json
+meta:
+  name: github.com/gocql/gocql
+  description: Package gocql implements a fast and robust Cassandra client for the Go programming language.
+  icon: at-symbol
+
+aspects:
+  - id: Replace with gocqltrace.CreateTracedSession
+    join-point:
+      # TODO: not implemented yet
+      method-call: github.com/gocql/gocql.Cluster.CreateSession
+    advice:
+      - wrap-expression:
+          imports:
+            gocql: github.com/gocql/gocql
+            gocqltrace: gopkg.in/DataDog/dd-trace-go.v1/contrib/gocql/gocql
+          template: |-
+            func() (*gocql.Session, error) {
+              return gocqltrace.CreateTracedSession({{ .Receiver }})
+            }()
```

Comment:
There's a few things that could be problematic here...

- The source position of the original call is lost here; and maybe we need to have a way to "fix that" (effectively stating that the `gocqltrace.CreateTracedSession` is actually standing in for the original call.
- The `{{ .Receiver }}` thing here would actually be something like `{{ .AST.Fun.X }}`
	- This is on account that the matched node would be a `*dst.CallExpr` of which the `.Fun` is a `*dst.SelectorExpr` of which we need to drop the selector only.